src/js/views/maps/ToolbarView.js

```diff
@@ -158,10 +171,15 @@ define(
             },
           },
           {
-            label: 'Home',
-            icon: 'home',
+            label: 'Reset',
+            icon: 'rotate-left',
             action: function (view, model) {
               model.flyHome();
+              
+              // Reset the visibility of all layers.
+              for (const layer of model.getAllLayers()) {
+                layer.set("visible", layer.get('originalVisibility'));
```

Comment:
seems like logic that should be handled within MapAsset (resetVisibility()?) rather than for Toolbar to fiddle with directly?

Reply:
➕ agree!

Reply:
Done